lexos/templates/scrub.html

```diff
@@ -2,355 +2,537 @@
-                } else {
-                    tagDict[this.name] = this.value || '';
-                }
-                                });
-           tagDict=JSON.stringify(tagDict);
-           console.log(tagDict);
-           $.ajax({
-                type: "POST",
-                url: "/xml",
-                data: tagDict,
-                contentType: 'application/json;charset=UTF-8',
-                cache: false,
-                beforeSend: function() {
-                    $("#xml-modal-status").show();
-                },
-                success: function(response){
-                    $("#xml-modal-status").hide();
-                    $('.modal.in').modal('hide');
-                },
-                error: function(jqXHR, textStatus, errorThrown){
-                    $("#error-modal .modal-body").html("Lexos could not perform the requested function.");
-                    $("#error-modal").modal();
-                    console.log("bad: " + textStatus + ": " + errorThrown);
-                }
-            });
-        });
-
-    });
-
-</script>
+    <!-- Latest compiled and minified DataTables CSS and JS -->
+    {% if config['LOCAL_MODE'] == True %}
+        <link rel="stylesheet" type="text/css"
+              href="{{ url_for('static', filename='node_modules/datatables.net-bs/css/dataTables.bootstrap.css') }}?ver={{ version }}"/>
+        <link rel="stylesheet" type="text/css"
+              href="{{ url_for('static', filename='node_modules/datatables.net-select-bs/css/select.bootstrap.css') }}?ver={{ version }}"/>
+        <script type="text/javascript"
+                src="{{ url_for('static', filename='node_modules/datatables.net/js/jquery.dataTables.js') }}?ver={{ version }}"></script>
+        <script type="text/javascript"
+                src="{{ url_for('static', filename='node_modules/datatables.net-select/js/dataTables.select.js') }}?ver={{ version }}"></script>
+        <script type="text/javascript"
+                src="{{ url_for('static', filename='node_modules/datatables.net-bs/js/dataTables.bootstrap.js') }}?ver={{ version }}"></script>
+    {% else %}
+        <link rel="stylesheet" type="text/css"
+              href="https://cdn.datatables.net/u/bs/jszip-2.5.0,pdfmake-0.1.18,dt-1.10.12,b-1.2.1,b-html5-1.2.1,b-print-1.2.1,fc-3.2.2,fh-3.1.2,se-1.2.0/datatables.min.css"/>
+        <script type="text/javascript"
+                src="https://cdn.datatables.net/u/bs/jszip-2.5.0,pdfmake-0.1.18,dt-1.10.12,b-1.2.1,b-html5-1.2.1,b-print-1.2.1,fc-3.2.2,fh-3.1.2,se-1.2.0/datatables.min.js"></script>
+        <script type="text/javascript"
+                src="{{ url_for('static', filename='node_modules/datatables.net-bs/js/dataTables.bootstrap.js') }}?ver={{ version }}"></script>
+    {% endif %}
+
+    <script type="text/javascript"
+            src="{{ url_for('static', filename='js/scripts_scrub.js') }}?ver={{ version }}"></script>
+    <script>
```

Comment:
Avoid scripts in HTML.

Reply:
I am not completely sure what this script does. I removed it and have done some testing and everything seems to still we working properly. I cannot find many of the elements that this script refers to on the page.

Reply:
Hum, talk to Weiqi about this.